prisma/schema.prisma

```diff
@@ -0,0 +1,92 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+enum Role {
+  ADMIN
+  USER
+}
+
+enum Visibility {
+  PUBLIC
+  PRIVATE
+  FRIENDS
+}
+
+model User {
```

Comment:
Maybe some kind of avatar for users?

Reply:
we can add integrate images later when we have some asset management like cloudinary

Reply:
yes for now i can also remove images from other columns as for now we don't have anything for storing assets.

Reply:
oh nevermind, i will leave it there, it won't bother anyone i guess